.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -25,22 +25,16 @@
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.20.15@sha256:6f2d011dffe182bad80b85f6c00e8ca9d86b5b8922cdf433d53575c4c5212248
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.21.12@sha256:f316b33dd88f8196379f38feb80545ef3ed44d9197dca1bfd48bcb1583210207
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.22.9@sha256:8135260b959dfe320206eb36b3aeda9cffcb262f4b44cda6b33f7bb73f453105
-    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.22.9@sha256:8135260b959dfe320206eb36b3aeda9cffcb262f4b44cda6b33f7bb73f453105
-      DEPLOYER_KIND_IP_FAMILY: ipv6
-    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.23.6@sha256:b1fa224cc6c7ff32455e0b1fd9cbfd3d3bc87ecaa8fcb06961ed1afb3db0f9ae
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.23.6@sha256:b1fa224cc6c7ff32455e0b1fd9cbfd3d3bc87ecaa8fcb06961ed1afb3db0f9ae
-      DEPLOYER_KIND_IP_FAMILY: ipv6
-    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.24.1@sha256:11276ce4763d5600379c0104c4cb51bf7420c8c02937708e958ad68d08e0910c
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.24.1@sha256:11276ce4763d5600379c0104c4cb51bf7420c8c02937708e958ad68d08e0910c
+    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
+    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.26.0@sha256:691e24bd2417609db7e589e1a479b902d2e209892a10ce375fab60a8407c7352
```

Comment:
nit: [1.26.2 is available](https://hub.docker.com/layers/kindest/node/v1.26.2/images/sha256-c39462fc9f460e13627cbd835b7d1268e4fd1a82d23833864e33ac1aaa79ee7a?context=explore)

Reply:
I will test it. Not sure it is compatible with Kind 0.15.0, which by the way could be updated.